Standards/CCSMB-5.md

```diff
@@ -0,0 +1,51 @@
+# CCSMB-5: PCM Audio Transmissions over modem
+*Author: Emma Beurskens <@emmaknijn>*
+*Version: v1.0.0*
+*Last updated: YYYY-MM-DD*
+
+## Design
+A client will only listen to packets while the audio is being transmitted from a server
+
+The server will communicate to the client over modem
+
+All servers must be on seperate channels
+
+PASC refers to "Protocol for Audio Streaming in Computercraft"
```

Comment:
I'm not sure whether to use "ComputerCraft" or "Computercraft here"

Reply:
do ComputerCraft, that seems to be common across references

Reply:
see https://github.com/dan200/ComputerCraft